webui/react/src/stores/cluster.tsx

```diff
@@ -28,12 +28,21 @@ const initClusterOverview: ClusterOverview = {
   [ResourceType.UNSPECIFIED]: structuredClone(initResourceTally),
 };
 
+const flexSchedulers: V1SchedulerType[] = [V1SchedulerType.PBS, V1SchedulerType.SLURM];
+
 /**
  * maximum theoretcial capacity of the resource pool in terms of the advertised
  * compute slot type.
  * @param pool resource pool
  */
 export const maxPoolSlotCapacity = (pool: ResourcePool): number => {
+  if (
+    flexSchedulers.includes(pool.schedulerType) &&
+    pool.slotsAvailable &&
```

Comment:
question: what is `pool.slotsAvailable &&` for? `pool.slotsAvailable` isnt `undefined` or `null`

Reply:
you're right; I removed that condition in latest commit
I think this maybe was written because `pool.slotsPerAgent` in the other conditional *can* be `undefined`